README.md

```diff
@@ -1 +1,160 @@
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://reactjs.org/">react</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Make simple math calculation**
+- **Generate random math-related quote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] Coming Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+- **ReactJs**
+- **NPM**
+
+In order to run this project you need:
+ 
+1. To have installed Nodejs in your Code editor. In case you don't have it, go to this <a href="https://nodejs.org/en/">site</a>  and download the setup based on your PC
+<p align="left">(<a href="https://nodejs.org/en/">install NodeJs</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder using the above command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/mrprotocoll/math-magician-microverse.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  npm install -y
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  cd my-project
+  npm start
+```
+
+Open <a href="http://localhost:3000/">http://localhost:3000/</a> to view it in the browser. The page will automatically reload if you make changes to the code. You will see the build errors and lint warnings in the console.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **mrprotocoll**
+
+- GitHub: [@mrprotocoll](https://github.com/mrprotocoll)
+- Twitter: [@dprotocoll](https://twitter.com/dprotocoll)
+- LinkedIn: [@mrprotocoll](https://www.linkedin.com/in/mrprotocoll)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add good UI/UX]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+https://github.com/mrprotocoll/math-magician-microverse
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please don't forget to follow me and give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great job working on this milestone and adding the correct ReadMe template, but unfortuntely, the MIT license link leads us to an error 404 page. It is not professional to have broken links pushed to our Github repository. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/62849872/218933130-a6f29c56-e15c-470e-83ba-f49c029dedcd.png)

Reply:
Thank You @t-yanick , i will make the changes and request a new review.